suite-common/wallet-core/src/transactions/transactionConstants.ts

```diff
@@ -0,0 +1 @@
+export const modulePrefix = '@common/wallet-core/transactions';
```

Comment:
Please scrap this file, we don't need files for one constant. Rename this constant to `actionPrefix` a move it to `transactionActions`, it belongs together anyway.